flowgate-api/src/main/java/com/vmware/flowgate/controller/AssetController.java

```diff
@@ -95,9 +95,9 @@ public HttpHeaders create(@RequestBody Asset asset) {
 
    @ResponseStatus(HttpStatus.CREATED)
    @RequestMapping(value = "/batchoperation", method = RequestMethod.POST, consumes = MediaType.APPLICATION_JSON_VALUE)
-   public void batchCreation(@RequestBody List<Asset> assets) {
+   public List<Asset> batchCreation(@RequestBody List<Asset> assets) {
```

Comment:
We should be careful of change the return. if the list is big, we may encounter issue. 

Do we have requirement to get the returned updated assets?  I remember that we only can get the assetID with couchbase database which is set by our code.